Formula/hbase.rb

```diff
@@ -1,9 +1,8 @@
 class Hbase < Formula
   desc "Hadoop database: a distributed, scalable, big data store"
   homepage "https://hbase.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=hbase/1.1.4/hbase-1.1.4-bin.tar.gz"
-  sha256 "2979384e7677f9ea95b8b2753d082b9a79cb9540a22dbcd072b942796363a925"
-  revision 1
+  url "https://www.apache.org/dyn/closer.cgi?path=hbase/1.2.1/hbase-1.2.1-bin.tar.gz"
```

Comment:
Is this stable? The upstream website says:

> We suggest downloading the current stable release.

Which links to `hbase-1.1.4-bin.tar.gz`